components/PageHeader.vue

```diff
@@ -1,32 +1,34 @@
 <template>
   <b-container
     fluid
-    class="border-bottom p-0 mb-3"
+    class="border-bottom mb-3 d-flex py-3 py-lg-1 px-lg-3 flex-column flex-lg-row"
+    data-qa="header"
   >
-    <b-navbar
-      class="pb-3 pt-3 py-lg-1 px-lg-3 flex-column flex-lg-row"
-      data-qa="header"
+    <header
+      class="col-lg-2 p-0 m-0 text-center text-lg-left navbar-brand d-flex align-items-center justify-content-center justify-content-lg-start"
+      role="banner"
```

Comment:
I think the search bar can go into the landmark "banner" too. At least, that it was I get from the documentation https://www.w3.org/TR/wai-aria-practices-1.1/#aria_lh_banner.

Reply:
sure I will put the search in the banner, just thought all landmarks had to be top-level but checked the docs and search doesn't have to be